client/actions/book_actions.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+
+import { ADD_BOOK, GET_ALL_BOOKS, GET_RENTED_BOOKS } from './types';
+
+const API_URL = 'http://localhost:8000/api/v1/books',
+  USER_API_URL = 'http://localhost:8000/api/v1/users';
+
+export function addNewBook(bookDetails) {
+  return dispatch => axios.post(`${API_URL}`, bookDetails)
+    .then((res) => {
+      dispatch({
+        type: ADD_BOOK,
+        message: res.data.message
+      });
+    })
+    .catch(error => error);
+}
+
+export function getAllBooks() {
+  return dispatch => axios.get(`${API_URL}`)
+    .then((res) => {
+      dispatch({
+        type: GET_ALL_BOOKS,
+        data: res.data
+      });
+      return res.data;
+    })
+    .catch(error => error);
+}
+
+export function deleteBook(bookId) {
+  return axios.delete(`${API_URL}/delete/${bookId}`)
```

Comment:
In this section, you're not only passing the variable `API_URL` but you're attaching a string url to it, so this is a good use case for a template literal `${}`. Template literals are string literals allowing embedded expressions. It's another way of doing string concatenation. You can read more about template literals and it's usage [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

Reply:
Thanks for the feedback